modules/builtin/src/content-types/image.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
     properties: {
       image: {
         type: 'string',
-        $subtype: 'media',
+        $subtype: 'image',
```

Comment:
@samuelmasse I haven't thought about it before, but this change can be breaking for some modules I think!